frontend/packages/kubevirt-plugin/src/k8s/requests/vm/create/default-vm.ts

```diff
@@ -26,6 +37,26 @@ export const resolveDefaultVM = async (params: DefaultVMLikeEntityParams): Promi
   const processedTemplate = await k8sCreate(ProcessedTemplatesModel, template.asResource(), null); // temporary
 
   const vm = new VMWrapper(selectVM(processedTemplate));
+  const cloudInitVolume = vm.getCloudInitVolume();
```

Comment:
this function already called resolveDefaultVMTemplate. So there is no need to repeat the logic here.